tests/kernel/install_ltp.pm

```diff
@@ -227,19 +227,9 @@ sub install_from_repo {
 sub setup_network {
     my $content;
 
-    $content = <<EOF;
-# ltp specific setup
-pts/1
-pts/2
-pts/3
-pts/4
-pts/5
-pts/6
-pts/7
-pts/8
-pts/9
-EOF
-    assert_script_run("echo \"$content\" >> '/etc/securetty'");
+    # pts in /etc/securetty
+    $content = '# ltp specific setup\npts/1\npts/2\npts/3\npts/4\npts/5\npts/6\npts/7\npts/8\npts/9\n';
```

Comment:
Maybe it's just me, but I find this less readable than before.

Reply:
@mnowaksuse yes, it was more readable before, but new lines in `script_run()` hangs the execution. It's a bug I was not able to solve, so this commit is a workaround.
I should probably document it somewhere or create a ticket to prevent others bump into this bug.

Reply:
I created https://progress.opensuse.org/issues/50945 and noted it in 2 related commits.